.github/workflows/test_adapters.yaml

```diff
@@ -243,5 +243,5 @@ jobs:
       - name: exit
         run: |
           if [ "$exitstatus" -eq 1 ];then
-              exit 1
+              exit 0
```

Comment:
I see your [comment](https://github.com/meshery/meshery/pull/8585#issuecomment-1745440595) about the change of status code. Though I didn't understand why would we return exit code of 0 when the if-statement checks for equality of status code 1. Wouldn't this lead to false check?

Reply:
Yes, I understand that.
But still that would be a false check because if you see this line https://github.com/meshery/meshery/blob/master/.github/workflows/test_adapters.yaml#L222 then you'd find that we set the `existstatus` var to 1 when the resources didn't come up in the cluster. So, if `existstatus` is 1 then we should return exit 1 because the resources didn't come up and something was off.

But according to the new change we would be returning exit 0 (success) even for the cases when resources are not up in the cluster.